src/interface/src/app/services/invites.service.ts

```diff
@@ -53,4 +53,15 @@ export class InvitesService {
       }
     );
   }
+
+  deleteInvite(planningAreaId: number, inviteId: number) {
+    return this.http.delete<Invite[]>(
+      BackendConstants.END_POINT.concat(
+        `/invites/invitations/planningarea/${planningAreaId}/${inviteId}`
```

Comment:
I'm still open to changing this endpoint format to just accept the inviteId, if we prefer. 
It's a simple change, obviously, I just want to be democratic about it :D

Reply:
Yeah, I agree. I updated these.